src/main/java/dk/bec/bookanything/dto/UserCreateDto.java

```diff
@@ -0,0 +1,35 @@
+package dk.bec.bookanything.dto;
+
+import dk.bec.bookanything.validator.ForeignKeyExistsConstraint;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Past;
+import javax.validation.constraints.Size;
+import java.time.LocalDate;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserCreateDto {
+
+    @Email
+    private String email;
+
+    @Size(min = 6)
```

Comment:
I saw that in user entity we do not have the same constraint. Please assure to have exact set of constraints on dto classes and referenced entities